cpp-package/src/data_filter.cpp

```diff
@@ -151,7 +151,7 @@ std::pair<BrainFlowArray<double, 2>, BrainFlowArray<double, 1>> DataFilter::get_
         throw BrainFlowException ("failed to compute the CSP filters", res);
     }
 
-    return std::make_pair (std::move (filters), std::move (output_eigenvalues));
+    return std::make_pair (filters, output_eigenvalues);
```

Comment:
just noting that here `std::move` does indeed have an effect in this case, because the values are passed to `std::make_pair` instead of returned directly.